Libplanet.Net/Consensus/ConsensusContext.cs

```diff
@@ -199,14 +202,17 @@ public void NewHeight(long height)
 
             if (!_contexts.ContainsKey(height))
             {
-                _contexts[height] = new Context<T>(
-                    this,
-                    _blockChain,
-                    height,
-                    _privateKey,
-                    _getValidators(height).ToList(),
-                    Step.Default,
-                    contextTimeoutOptions: _contextTimeoutOption);
+                lock (_contextLock)
```

Comment:
This should be moved one layer up I think?

Reply:
https://github.com/planetarium/libplanet/blob/ab82ef9d1219224f7d877fb27b45004a6e7e1e36/Libplanet.Net/Consensus/ConsensusContext.cs#L206-L223

Amended.